com.jianguoyun.Nutstore.appdata.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.jianguoyun.Nutstore</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <name>Nutstore</name>
+  <name xml:lang="zh-cn">坚果云</name>
+  <summary>Share your files anytime, anywhere, with any device</summary>
+  <summary xml:lang="zh-cn">任何设备，随时随地实现文件共享</summary>
+  <description>
+    <p>
+      Multi folder sync, Intelligent incremental sync, Lightning download, Automatic file lock, Quality apps integration: Continuous innovation, truly useful!
+    </p>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Shanghai YiCun Network Tech Co., Ltd.</p>
+  </description>
+  <description xml:lang="zh-cn">
+    <p>
+      只需将您的个人文件放入“我的坚果云”文件夹，或者将您的文件夹用坚果云同步，您就可以便捷地访问自己的文件，并安全地保存它们。使用云存储，就是这么简单！
+    </p>
+    <p>
+      注意：本flatpak打包版本没有得到上海亦存网络科技有限公司的认证和支持。
+    </p>
+  </description>
+  <categories>
+    <category>Network</category>
+    <category>Office</category>
+    <category>Utility</category>
+  </categories>
+  <screenshots>
+    <screenshot type="default">
+      <image>http://help.jianguoyun.com/wp-content/uploads/2014/02/chenshufei_1465781497542_28.png</image>
+    </screenshot>
+    <screenshot>
+      <image>http://help.jianguoyun.com/wp-content/uploads/2011/03/%E7%BD%91%E9%A1%B5%E7%AB%AF%E8%8E%B7%E5%8F%96%E5%88%86%E4%BA%AB%E9%93%BE%E6%8E%A5.gif</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://www.jianguoyun.com/</url>
+  <developer_name>Shanghai YiCun Network Tech Co., Ltd</developer_name>
+  <content_rating type="oars-1.1" >
+    <content_attribute id="social-chat">moderate</content_attribute>
+    <content_attribute id="social-info">mild</content_attribute>
+    <content_attribute id="money-purchasing">intense</content_attribute>
+  </content_rating>
+  <releases>
+    <release type="stable" date="2019-01-29" version="4.2.2">
+    	<description>
+    		<p>修复在部分系统中托盘图标显示过大的问题</p>
```

Comment:
The p tags are translatable https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-description, it would be nice to have the default in English and add a translation for Chinese